airbyte-integrations/connectors/source-bamboo-hr/source_bamboo_hr/manifest.yaml

```diff
@@ -0,0 +1,274 @@
+        request_headers:
+          accept: application/json
+        request_body_json: {}
+        request_parameters: {}
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - employees
+    primary_key:
+      - id
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        type: object
+        $schema: http://json-schema.org/schema#
+        properties:
+          id:
+            type: string
+          division:
+            type:
+              - 'null'
+              - string
+          jobTitle:
+            type:
+              - 'null'
+              - string
+          lastName:
+            type: string
+          linkedIn:
+            type:
+              - 'null'
+              - string
+          location:
+            type: string
+          photoUrl:
+            type: string
+          firstName:
+            type: string
+          instagram:
+            type:
+              - 'null'
+              - string
+          workEmail:
+            type:
+              - 'null'
+              - string
+          workPhone:
+            type:
+              - 'null'
+              - string
+          department:
+            type: string
+          supervisor:
+            type: string
+          displayName:
+            type: string
+          mobilePhone:
+            type:
+              - 'null'
+              - string
+          photoUploaded:
+            type: boolean
+          preferredName:
+            type:
+              - 'null'
+              - string
+          canUploadPhoto:
+            type: number
+          workPhoneExtension:
+            type:
+              - 'null'
+              - string
+  - name: custom_reports_stream
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      paginator:
+        type: NoPagination
+      requester:
+        path: reports/custom
+        type: HttpRequester
+        url_base: https://api.bamboohr.com/api/gateway.php/{{ config['subdomain'] }}/v1/
+        http_method: POST
+        authenticator:
+          type: BasicHttpAuthenticator
+          password: ''
+          username: '{{ config[''api_key''] }}'
+        request_headers: {}
+        request_body_json:
+          title: Airbyte
+          fields: '{{ config[''custom_reports_fields''] }}'
+        request_parameters:
+          format: JSON
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - employees
+    primary_key:
+      - id
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        type: object
+        $schema: http://json-schema.org/schema#
+        properties:
+          id:
+            type: string
+          lastName:
+            type: string
+          bestEmail:
+            type: string
+          firstName:
+            type: string
```

Comment:
i don't think this is the correct schema for a custom report. Can you ask in slack about how to have dynamic schema in low code?

Reply:
Another comment here missing reply.

Reply:
Yes, that's right
I asked in slack but no reply, as far as i read documentation it is notcurrrently possible to have dynamic schema in no-code